editor/project_manager/project_dialog.cpp

```diff
@@ -240,22 +239,16 @@ void ProjectDialog::_validate_path() {
 }
 
 String ProjectDialog::_get_target_path() {
+	String path;
 	if (mode == MODE_NEW || mode == MODE_INSTALL) {
-		return project_path->get_text();
+		path = project_path->get_text();
 	} else if (mode == MODE_IMPORT) {
-		return install_path->get_text();
-	} else {
-		ERR_FAIL_V("");
+		path = install_path->get_text();
 	}
-}
-void ProjectDialog::_set_target_path(const String &p_text) {
-	if (mode == MODE_NEW || mode == MODE_INSTALL) {
-		project_path->set_text(p_text);
-	} else if (mode == MODE_IMPORT) {
-		install_path->set_text(p_text);
-	} else {
-		ERR_FAIL();
+	if (project_name_suffix->is_visible() || install_name_suffix->is_visible()) {
+		path = path.path_join(auto_dir);
 	}
```

Comment:
```suggestion
	if (project_name_suffix->is_visible()) {
		path = path.path_join(project_name_suffix->get_text());
	} else if (install_name_suffix->is_visible()) {
		path = path.path_join(install_name_suffix->get_text());
	}
```
Use get_text to get the current value, which may be different from auto_dir